### PR TITLE
New homepage author ad design

### PR DESCRIPTION
**Pull request by maxkfranz**
- This change revises the homepage with an author-centric ad.
- The page is separated into several sections:  The top banner introduces the project name with the logo and the call-to-action button.  The banner background image is of a conference, to reference that the project is about community and information sharing.
- Each section concisely describes the verbs in the tagline "summarise, share, discover".  Each section has an associated background image to reinforce the concept of the section and to keep things visually interesting.  These sections condense the text from prior designs such that extra sections like FAQs are unnecessary.
- The new logo and icon are used in the homepage.
- There is a second call-to-action button at the bottom of the page, to make it easy for the author to request an invitation after he's read the whole page.
- Institution branding is included at the bottom of the page.
- We could iterate on the copy and the banner images a bit, but I think they're pretty well good enough for now.  The "Discover" section, for example, could be improved once we have better support for data exploration.

Ref : Landing page (revisited) #534

--

Desktop preview:

Desktop - top</img>

Desktop - middle</img>

Desktop - middle 2</img>

Desktop - bottom</img>

Desktop - call-to-action</img>

Desktop - call-to-action gif</img>

Mobile preview:

Mobile - top</img>

Mobile - middle</img>

Mobile - bottom</img>

### CODE REVIEW

**Comment by maxkfranz** [comment]
Note: The CTA button does not send any emails yet, as this needs to be coordinated with the new admin panel.  The operations are stubbed out and simulated for now for easy updating in future.  The CTA UI itself works, so it's sufficient for UI testing.

File: src/client/components/home.js [hunk 1]
```diff
@@ -1,9 +1,202 @@
...
+        h('div.home-credits', [
+          h('p.home-credit-logos', [
+            h('i.home-credit-logo'),
+          ]),
+          h('p.home-credit-text', [
+            'Biofactoid is freely brought to you in collaboration with ',
+            h('a.plain-link', { href: 'https://baderlab.org' }, 'Bader Lab at the University of Toronto'),
+            ', ',
+            h('a.plain-link', { href: 'https://sanderlab.org' }, 'Sander Lab at Harvard'),
+            ', and the ',
```
Comment (jvwong) [inline src/client/components/home.js]: Sander lab site only uses `http` (http://sanderlab.org/)

File: src/client/components/home.js [hunk 2]
```diff
@@ -1,9 +1,202 @@
...
+            h('a.plain-link', { href: 'https://sanderlab.org' }, 'Pathway and Omics Lab at the University of Oregon'),
+            '.  ',
```
Comment (jvwong) [inline src/client/components/home.js]: I'm not completely sure where Emek wants us to point this.

Comment (maxkfranz) [reply src/client/components/home.js]: Let's just link to the university's homepage until he gives us an alternate link.

File: src/client/components/home.js (hunk 1 above)
Comment (maxkfranz) [reply src/client/components/home.js]: OK

**Comment by maxkfranz** [comment]
Let's consider copy changes and article handling for other PRs.  Let's merge this in once any bugs we find are addressed.

**Comment by maxkfranz** [comment]
Re. Pubmed lookup: You can get the ID pretty easily using the webservice.  Example request for your example article:

https://eutils.ncbi.nlm.nih.gov/entrez/eutils/esearch.fcgi?db=pubmed&term=PTEN%20Suppresses%20Glycolysis%20by%20Dephosphorylating%20and%20Inhibiting%20Autophosphorylated%20PGK1

Basically just pass the article title and you get the ID.  If article titles are sufficiently unique, maybe it's fine to just let the user paste in his article's title.

**Comment by maxkfranz** [comment]
I'm merging this in for now so it's easier for everyone to try it out on the CI server.  Let's address tweaks in another PR.

**Pull request merged**